server/pulp/server/constants.py

```diff
@@ -1,6 +1,16 @@
+from pulp.server.config import config
+
 LOCAL_STORAGE = "/var/lib/pulp/"
 
 PULP_USER_METADATA_FIELDNAME = 'pulp_user_metadata'
 PULP_DJANGO_SETTINGS_MODULE = 'pulp.server.webservices.settings'
 PULP_STREAM_REQUEST_HEADER = 'Pulp-Stream-Request'
 SUPER_USER_ROLE = 'super-users'
+
+# The amount of time (in seconds) between process wakeups to "heartbeat" and perform
+# their tasks.
+PULP_PROCESS_HEARTBEAT_INTERVAL = int(config.getint('tasks', 'worker_failover_time') / 5)
```

Comment:
This is probably ok as-is, but maybe it would be good to add a brief explanation behind the math and magic numbers going on, the distinction between worker_failover_time and the process_timeout_interval, etc.

I'm fine without it too, though.

Reply:
I think a link to the redmine comments where this is discussed would be a good way to help us (and others) remember.

Reply:
+1 on that